enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -88,6 +89,9 @@ export const DEFAULT_METABASE_COMPONENT_THEME: MetabaseComponentTheme = {
       label: { fontSize: FONT_SIZES.goalLabel.px },
     },
   },
+  popover: {
+    zIndex: OVERLAY_Z_INDEX,
+  },
```

Comment:
If we already have the default z-index value defined via CSS variables, then there is no need to re-define them here in `defaultComponentTheme` as it should inherit from CSS root.